### PR TITLE
Suggesting multiple changes for better readability

### PR DESCRIPTION
**Pull request by Mohit-Nain**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/sources/features/datasources/mssql.md [hunk 1]
```diff
@@ -293,7 +293,7 @@ GROUP BY
 ORDER BY 1
 ```
 
-When above query are used in a graph panel the result will be two series named `Metric A` and `Metric B` with a sum of `valueTwo` plotted over `time`.
+When above query is used in a graph panel the result will be two series named `Metric A` and `Metric B` with a sum of `valueTwo` plotted over `time`.
```
Comment (oddlittlebird) [inline docs/sources/features/datasources/mssql.md:296]:
```suggestion
When the above query is used in a graph panel, the result is two series named `Metric A` and `Metric B` with a sum of `valueTwo` plotted over `time`.
```

File: docs/sources/features/datasources/mssql.md [hunk 2]
```diff
@@ -444,7 +444,7 @@ ORDER BY 1
 
 ## Stored procedure support
 
-Stored procedures have been verified to work. However, please note that we haven't done anything special to support this why there may exist edge cases where it won't work as you would expect.
+Stored procedures have been verified to work. However, please note that we haven't done anything special to support this so there may exist edge cases where it won't work as you would expect.
```
Comment (oddlittlebird) [inline docs/sources/features/datasources/mssql.md:447]:
```suggestion
Stored procedures have been verified to work. However, please note that we haven't done anything special to support this, so there might be edge cases where it won't work as you would expect.
```

File: docs/sources/features/datasources/mssql.md [hunk 3]
```diff
@@ -491,7 +491,7 @@ We can define a stored procedure that will return all data we need to render 4 s
 In this case the stored procedure accepts two parameters `@from` and `@to` of `int` data types which should be a timerange (from-to) in epoch format
 which will be used to filter the data to return from the stored procedure.
 
-We're mimicking the `$__timeGroup(time, '5m')` in the select and group by expressions and that's why there's a lot of lengthy expressions needed -
+We're mimicking the `$__timeGroup(time, '5m')` in the select and group by expressions and that's why there are a lot of lengthy expressions needed -
```
Comment (oddlittlebird) [inline docs/sources/features/datasources/mssql.md:494]:
```suggestion
We're mimicking the `$__timeGroup(time, '5m')` in the select and group by expressions, and that's why there are a lot of lengthy expressions needed -
```

**Review by oddlittlebird — CHANGES_REQUESTED** [review]

**Review by aknuds1 — APPROVED** [review]

**Review by aknuds1 — APPROVED** [review]

**Review by oddlittlebird — APPROVED** [review]
LGTM. Thanks for improving this doc!

**Pull request merged**